docs/source/install.rst

```diff
@@ -5,7 +5,15 @@ Building and Installing
 Prerequisites
 =============
 
--  A ROCm enabled platform, more information `here <https://rocm.github.io/>`_.
+* If using the rocBLAS backend on an AMD machine:
+
+  * A ROCm enabled platform, more information `here <https://rocm.github.io/>`_.
+  * A compatible version of rocBLAS
+
```

Comment:
Do you want to mention, optionally a compatible version of rocSOLVER for more functionality